tests/cbor_encode/test1_suit/src/main.c

```diff
@@ -6,22 +6,7 @@
 
 #include <ztest.h>
 #include "manifest3_decode.h"
-
-
-bool cbor_encode_SUIT_Outer_Wrapper(
-		const uint8_t * p_payload, size_t payload_len,
-		SUIT_Outer_Wrapper_t * p_result,
-		size_t *p_payload_len_out);
-
-bool cbor_encode_SUIT_Command_Sequence(
-		const uint8_t * p_payload, size_t payload_len,
-		SUIT_Command_Sequence_t * p_result,
-		size_t *p_payload_len_out);
-
-bool cbor_encode_SUIT_Authentication_Wrapper(
-		const uint8_t * p_payload, size_t payload_len,
-		SUIT_Authentication_Wrapper_t * p_result,
-		size_t *p_payload_len_out);
+#include "manifest3_encode.h"
```

Comment:
Does this has anything to do with this patch? Not complaining, just trying to figure out if I understood the change :D

Reply:
Yes, it's a good question :). This patch makes it possible to include both the encoder and decoder header files, since the typedefs are in a separate (shared) file. Before this patch, the compiler would complain that the types were duplicated. Note that to do this, you must manually specify the typedef header file's name, since by default, encoding and decoding get different names. This is done in CMakeLists.txt above.